### ISSUE TITLE
[Bug]: LogSender Disconnecting

### ISSUE THREAD

**Issue by CyberXCoder-6649** [AndroidIDEOfficial/AndroidIDE#1385]
### What happened?

When trying to inspet logs no logs found when I checked IDE Logs its says Disconnecting but when i opens app instantly it works i think maybe because of any timeout or something if ide has timeout for Log Sender

What i understand about issue i mentioned above else you know

### What version of AndroidIDE you're using?

v2.5.3 (debug builds)

### Relevant log output

```shell
LogReceiverImpl           D   Disconnecting from all senders... 
LogReceiverImpl           I   Disconnecting from client: 'com.test' 
LogReceiverImpl           I   Total clients connected: 0 
AppLogFragment            I   Unbound from LogReceiver service
```

### Duplicate issues

- [X] This issue has not been reported yet.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by sb-codz** [comment]
@itsaky 
I have a suggestion for app logging. Though it may seem like an old method, I've been using it for a few days. The method is as follows: the app's log file will be stored in the app's external file directory, and the app will create a log file to be read within the Android IDE.

This approach ensures that server-side logging won't be necessary, leading to a reduction in system lag. While I am reading the log file using the File Manager.

By reading this, you can surely understand what will happen.
```java
File file = new File(mContext.getExternalFilesDir(null), "app_logs.txt");

**Comment by itsaky** [comment]
A fix was added in commit 3f15af6. Please checkout out the [corresponding CI build](https://github.com/AndroidIDEOfficial/AndroidIDE/actions/runs/7278437272).
